index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="ko-KR">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Starbucks</title>
+  <link rel="stylesheet" href="./reset.css" />
+  <link rel="stylesheet" href="./index.css" />
+</head>
+<body> 
+  <container class="container"> 
+  <main class="main"> 
+    <h2 class="a11yHidden">main_coffee</h2>
+    <section class="main_coffee"> 
+      <div class="first_item"><img src="./2022_Autumn_main_title (2).png" alt="#" /></div>
+      <div class="second_item"><img src="./2022_Autumn_main_drink_02.png" alt="#" /></div>
+      <div class="third_item"><img src="./2022_Autumn_main_drink_01.png" alt="#" /></div>
+      <div class="fourth_item"><img src="./2022_Autumn_main_drink_03.png" alt="#" /></div>
```

Comment:
Try using nth-child selectors instead of giving different class names for the drink items. Think about why I would suggest such method and when and why it would be better to use this method.